home/migrations/0102_remove_whatsapptemplate_name_whatsapptemplate_slug.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.2.20 on 2025-05-28 12:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("home", "0101_alter_whatsapptemplate_locale"),
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(
+            model_name="whatsapptemplate",
+            name="unique_name_locale",
+        ),
+        migrations.RemoveField(
+            model_name="whatsapptemplate",
+            name="name",
+        ),
+        migrations.AddField(
+            model_name="whatsapptemplate",
+            name="slug",
+            field=models.SlugField(
+                default="",
+                help_text="A unique identifier for this WhatsApp Template",
+                max_length=255,
+            ),
+        ),
+       
+        migrations.AddConstraint(
+            model_name="whatsapptemplate",
+            constraint=models.UniqueConstraint(
+                fields=("slug", "locale"), name="unique_slug_locale"
+            ),
+        ),
```

Comment:
I think this migration will fail. Since we're removing and then adding a field with a default value of "", that means that all slugs will just be empty strings, and so there will definitely be duplicates.

What we need to do instead is first add the slug field, make the default value a slugified version of the name field, and then remove the name field, and then add the constraint.

Reply:
Cool, have updated